packages/client-app/src/components/ActionModal/ActionModal.vue

```diff
@@ -53,7 +53,7 @@ const changeTab = (index: number) => {
     <div
       class="animate-modal-fade bg-backdrop fixed left-0 top-0 z-[1001] h-screen w-screen p-[20px] opacity-0">
       <DialogPanel
-        class="animate-modal-pop before:bg-b-1 relative mx-auto mt-20 w-full max-w-[480px] scale-[0.98] rounded-lg opacity-0 before:absolute before:z-0 before:block before:h-full before:w-full before:rounded-lg before:content-['']">
```

Comment:
@cameronrohani I moved this to a `<div>` at first but it didn't seem to be doing anything so I removed it